src/app/loadout/mod-utils.ts

```diff
@@ -89,22 +89,20 @@ export function isInsertableArmor2Mod(
 }
 
 /**
- * Generates a unique key for a mod when rendering. As mods can appear multiple times as
- * siblings we need to count them and append a number to its hash to make it unique.
- *
- * Note that counts is mutated and a new object should be passed in with each render.
+ * Supplies a function that generates a unique key for a mod when rendering.
+ * As mods can appear multiple times as siblings we need to count them and append a
+ * number to its hash to make it unique.
  */
-export const getModRenderKey = (
-  mod: PluggableInventoryItemDefinition,
-  /** A supplied object to store the counts in. This is mutated. */
-  counts: Record<number, number>
-) => {
-  if (!counts[mod.hash]) {
-    counts[mod.hash] = 0;
-  }
+export function useGetModRenderKey() {
```

Comment:
I don't know that this is really a hook since it doesn't use hooks itself.

Reply:
Yeah it could be by placing the count object inside a ref but I am not sure that is actually necessary due to each mod purely being a view component with no state. Basically it just felt hookish, but I will rename the function.